yoyodyne/models/modules/linear.py

```diff
@@ -0,0 +1,44 @@
+"""Linear module class."""
+
+from ... import data
+from . import base
+
+import torch
+from torch import nn
+
+
+class LinearEncoder(base.BaseModule):
+    """Embeds the input tensor, and then applies a affine projection.
+
+    This produces a simple non-contextual encoding of the input tensor.
+
+    Args:
+        bidirectional (bool).
+        *args: passed to superclass.
+        **kwargs: passed to superclass.
+    """
+
+    linear: nn.Linear
+
+    def __init__(self, *args, output_size: int, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.linear = nn.Linear(self.embedding_size, output_size)
```

Comment:
add flag for toggling the linear bias. (there's some random edge papers that explore adding bias to features so easier if we keep as a secret argument

Reply:
That's my point, just add a default `bias=True`. Never have to mess with it but don't have to remember that default.